docs/can-guides/commitment/recipes/playlist-editor/3-search.js

```diff
@@ -1,6 +1,6 @@
-var PlaylistVM = can.DefineMap.extend("PlaylistVM", {
+const PlaylistVM = can.DefineMap.extend("PlaylistVM", {
   init: function() {
-    var self = this;
+    const self = this;
```

Comment:
can we use arrow functions and remove this?

Reply:
IE11 doesn’t support arrow functions; I can be persuaded that the code in our guides doesn’t have to support IE11, but it’d take longer for me to update the code and make sure the highlighting is correct, so I’m not going to make this change now.